spec/lib/publishing_api_payload_spec.rb

```diff
@@ -18,6 +18,70 @@
       expect(payload).not_to include(:first_published_at)
     end
 
+    it "includes a public_updated_at timestamp for draft editions" do
+      freeze_time do
+        document_type = build(:document_type)
+        edition = build(:edition, document_type: document_type)
+
+        payload = described_class.new(edition).payload
+
+        payload_hash = {
+          public_updated_at: Time.current,
+        }
+
+        expect(payload).to match a_hash_including(payload_hash)
+        expect(payload).not_to include(:first_published_at)
+      end
+    end
+
+    it "includes a first_published_at and public_updated_at timestamp for published editions" do
+      freeze_time do
```

Comment:
I wonder if you can avoid the need for freeze time here by setting fixed dates? I assume we're more testing it passes dates on rather than it is the current time.

Reply:
I've removed the freeze_time from all the tests apart from those where we expect the current time